onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selector_action_transformer.cc

```diff
@@ -46,26 +48,49 @@ void DropQDQNodesRules(SelectorActionRegistry& qdq_selector_action_registry) {
 
   std::unique_ptr<Action> drop_action_no_int16 = std::make_unique<MergeIntoTargetFixed>(
       std::vector<NodeAndMoveInfo>(moves));  // Copy before std::move(moves)
+  std::unique_ptr<Action> drop_action_no_nonpositive_scale = std::make_unique<MergeIntoTargetFixed>(
```

Comment:
Can we avoid the double negative any maybe have `drop_action_has_positive_scale`?

Reply:
I think this action ended up being removed